examples/slackcontroller/slackcontroller.go

```diff
@@ -23,8 +23,10 @@ import (
 	"time"
 
 	"github.com/golang/glog"
+	"github.com/pkg/errors"
+
 	corev1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/api/errors"
+	api_errors "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
s/api_errors/apierrors

This can keep consistent with Kubernetes source code.

Reply:
@gyliu513 done.